Reference Topic: tasks-get-task-by-due-date.md (#48)
UWC2-APIDOC/to-do-service-public: docs/api/tasks-get-task-by-due-date.md

---
layout: page
---

# Get task by due date

> Returns an array of  [`task`](task) objects that contains only the task specified by the `due_date` parameter, if it exists.

Reviewer: ```suggestion
Returns an array of  [`task`](task) objects with the specified `due_date`, if any exist.
```
To shorten and match the example shown below.

## URL

```shell

> {server_url}/tasks/{due_date}

Reviewer: ```suggestion
{server_url}/tasks?due_date=<date_string to match>
```
Does this need a query parameter (like the suggestion) instead of  URL parameter?
